testsuite/classlibrary/TestSoundFile_Server.sc

```diff
@@ -0,0 +1,48 @@
+TestSoundFile_Server : UnitTest {
+
+	var server, soundFile;
+
+	setUp {
+		server = Server(this.class.name);
+		this.bootServer(server);
+
+		soundFile = SoundFile(Platform.resourceDir +/+ "sounds/a11wlk01.wav");
+	}
+
+	tearDown {
+		soundFile.close;
+		Buffer.freeAll;
+		server.quit.remove;
+	}
+
+	test_cue_passInEvent {
+		var cueEvent, testEvent, condition = Condition.new;
+
+		testEvent = (foo: true);
+		cueEvent = soundFile.cue(testEvent);
+		this.assert(cueEvent[\foo], "Cue should take an Event as first argument");
+	}
+
+	test_cue_playNow {
+		var cueEvent, condition = Condition.new;
+
+		OSCFunc({ condition.unhang }, '/n_go', server.addr);
+		cueEvent = soundFile.cue((), true);
+		fork { 3.wait; condition.unhang };
```

Comment:
stop this route at the end of the test so it doesn't fire and unhang a future hang